packages/import-bundle/src/compartment-wrapper.md

```diff
@@ -45,11 +45,12 @@ function getOdometer() {
 }
 
 const inescapableTransforms = [milageTransform];
-const inescapableGlobalLexicals = { getOdometer };
-const WrappedCompartment = wrapInescapableCompartment(Compartment,
-                                                      inescapableTransforms,
-                                                      inescapableGlobalLexicals);
-const endowments = {};
+const WrappedCompartment = wrapInescapableCompartment(
```

Comment:
note: the prose at the beginning of this file references the feature being removed, search for `provide additional global lexicals to its environment` and maybe remove it

Reply:
In fact, I think this example needs rethought.. the imposed transform will emit code that still works (it tries to call `getOdometer()`, which still works, but as a global global, not a global-lexical), but the user code can use `globalThis['get'+'Odometer']().read()`, which it could not do before

We don't need that restriction in agoric-sdk anymore, and I totally support removing it, but maybe the example we use here should emphasize the transform without also mentioning a confinement behavior that no longer exists.